xml/admin_ceph_gateway.xml

```diff
@@ -288,12 +288,16 @@ for bucket in conn.get_all_buckets():
      you more about its command line options.
     </para>
     <para>
-     The package is included in the 'Public Cloud' module for &sle; 12 SP3.
+     The package is included in the 'Public Cloud' module for &sle; 12 since SP3 and
+     &sle; 15.
      Before installing the package, you need to activate the module and refresh
      the software repository:
     </para>
 <screen>&prompt.root;SUSEConnect -p sle-module-public-cloud/12/x86_64
 sudo zypper refresh</screen>
+     <para>Or</para>
+<screen>&prompt.root;SUSEConnect -p sle-module-public-cloud/15/x86_64
+sudo zypper refresh</screen>     
```

Comment:
This looks inconsistent to me, myabe just a heritage, but: in one &lt;screent> element, you start with root prompt, while on its 2nd line, you use `sudo`
```suggestion
&prompt.root;zypper refresh</screen>     
```